wordpress-mobile/WordPress-iOS
# Update to latest version of Aztec (1.0.0 bet 5)

This PR updates to latest Aztec pod

To test:
 - Run the editor using Aztec
 - Check if all is working ok

Needs review: @jleandroperez
